seahub/templates/shared_file_view.html

```diff
@@ -9,9 +9,17 @@
         margin-bottom:12px;
     }
 </style>
+<link rel="stylesheet" type="text/css" href="{% static "css/eyeos.css" %}" />
 {% endblock %}
 
 {% block main_panel %}
+    <div id='publicMessage'>
+        {% trans "File shared using <strong>Open365</strong> personal cloud. " %}
```

Comment:
please remove trailing spaces from the string to be translated (see comment in the .po files, too).